src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -13,10 +13,11 @@ namespace System.Xml.Serialization
     using System.Threading;
     using System.Globalization;
     using System.Security;
-    using System.Security.Policy;
     using System.Xml.Serialization.Configuration;
     using System.Diagnostics;
+#if CODEDOM
```

Comment:
I first added #ifdef in the code, but then I decided to remove them completely to keep code clean. I will remove this one as well to keep it consistent.